utils/utils.go

```diff
@@ -97,3 +98,8 @@ func DisplayURL(urlStr string) string {
 	}
 	return u.Hostname() + u.Path
 }
+
+// Maximum length of a URL: 8192 bytes
+func ValidURL(urlStr string) bool {
+	return bytes.NewReader([]byte(urlStr)).Len() < 8192
```

Comment:
I think `len()` would do the job
```suggestion
	return len(urlStr) < 8192
```